### ISSUE TITLE
Breakpoints: space to check / uncheck checboxes looses keyboard focus

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52299]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.4.3
Tool Used : Keyboard

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "View Debug "(Cntrl+Shift+D) button.
3. Navigate to "All Exceptions" and "User Handled Exceptions" checkboxes in "Breakpoints" section.
4. Check/Uncheck checkboxes and use "Tab" each time to focus them.

**Actual:**
Keyboard focus does not remain on the checkbox when any state is changed. "Tab" is needed to bring back checkbox on focus for first checkbox and for second several tabs are used to bring back focus.

**Expected:**
Keyboard focus should remain on the checkbox on which any state change action is performed.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will find difficulty in navigating between these checkboxes since they would need to "Tab" again and again to bring focus back to same control once their states are changed.

**MAS Reference:**
MAS 2.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_Keyboard_CheckboxFocus.pptx](https://github.com/Microsoft/vscode/files/2114683/A11y_VSCode_ViewDebug_Keyboard_CheckboxFocus.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@bpasero @joaomoreno ping for ideas and help

**Comment by isidorn** [comment]
@bpasero thanks for trying
@bvancea might you know? :)

**Comment by isidorn** [comment]
Ok after some investigation it seems like the issue is [here](https://github.com/Microsoft/vscode/blob/646d6fc6b8dd643d8c1401677eb3783e73db72f3/src/vs/base/browser/ui/list/listWidget.ts#L359)

The list queries for the element that has the `tabIndex` property however it should respect if tabIndex is set to -1.
@joaomoreno I have pushed 7a629db352 to address this
